docs/QuickStart.md

```diff
@@ -0,0 +1,58 @@
+## Getting Started / Quick Start
+
+### Features
+
+* Parses Mustache templates and an extended template format (in YAML)
+* Generates a view schema from parsed template data
+* Supports Mustache partials and sections
+* Renders templates with user-provided views
+* Validates user-provided views against generated view schema
```

Comment:
"view(s)" -> "parameter(s)"

Reply:
I've updated this in the README #features.